packages/models-library/src/models_library/projects.py

```diff
@@ -200,6 +200,7 @@ class Config:
 
 
 class Owner(BaseModel):
+    user_id: int = Field(..., description="Owner user id", example=["2"])
```

Comment:
This library is intended to host models that can be reused entirely by  for schemas or domain models. That said, it is not a good idea to add too many restrictions. 

E.g. if you use this model for data coming from a user, probably the ``user_id`` is NOT required since it is the system that defines it. And the other way around, if it is used to model data coming from the system, sometimes you might want it required. NOTE that this library is intended to be used a **base** for both so I would not add too restrictive rules. In any case, this does NOT have an effect on the front-end since it does not use any kind of schema validator, right?

 Moreover, an owner might not be registered (for some reason) in the user's table, therefore this might be set as ``null``.


**Another thing**. I would avoid all trivial examples and descriptions and rather use this time to add meaninful programmatic [contraints](https://pydantic-docs.helpmanual.io/usage/types/#constrained-types) For instance, it is more meaningful for the pydantic model to define that ``user_id`` is an non-negative integer with any other extra property that it might have. 
Example

```python
class Owner(BaseModel):
       user_id: Optional[ conint(ge=0) ] = Field(None, description="Owner's identifier (if any) when registered in the user's database table")
```
This description allows us to do more things programatically e.g.  automatically generate fake values, validate more accurately your definitions, load automaticaly

Reply:
It is not optional, it is actually more important than the other two fields. I just extended what we already had.